src/Http/Controllers/Api/DiscussionController.php

```diff
@@ -88,7 +88,7 @@ public function store(StoreDiscussionRequest $request)
      */
     public function show($id)
     {
-        $discussion = Discussion::where('id', $id)
+        $discussion = Discussion::where('id', is_numeric($id) ? (int)$id : $id)
```

Comment:
Right, so this that returns the $id both times is fine? @Lukasz93P

Reply:
I don't understand your question, is totally different because one column is integer while other is string.

Reply:
Can we do `is_numeric($id) ? (int)$id : **$id**` on the other changes as well?

Reply:
Ech...My bad i didn't noticed...I wanted to type is_numeric($id) ? (int)$id : 0 ;)
I will fix this in few minutes.

Reply:
If migrations generated by this package creates migrations with integers as ids, then nobody should change this and use string ids...

Reply:
HAha already forgot how I made it. Still think is more natural to return $id instead of a 0

Reply:
@sbarbat look now, I have changed this, now it will work for both cases.

Reply:
@sbarbat tell me what do you think about it now.
If it's ok, then you can expect more pull requests from me because I have an interest in adding some small adjustments to this package.

Reply:
Perfect, thanks mate! Yeah if you can contribute would be great! I'm not having much spare time to spend updating it. But will try to make some to do so

Reply:
@sbarbat my plan is to allow custom user avatar url resolver, because my app does not store urls in users table - it receives them from another application.